packages/featureflags/FeatureFlagsContext.tsx

```diff
@@ -40,7 +40,10 @@ const FeatureFlagsProviderWrapper = ({ children }) => {
                     })
                     .catch(e => console.error(e))
             }
-        }, FEATURES_RE_FETCH_INTERVAL)
+        }
+
+        fetchFeatures()
+        const handler = setInterval(() => fetchFeatures(), FEATURES_RE_FETCH_INTERVAL)
```

Comment:
Why?

Reply:
The first time the callback will be executed after the delay. And feature flags will be received only after a certain time